packages/expo-modules-core/android/src/main/java/expo/modules/kotlin/jni/CppType.kt

```diff
@@ -0,0 +1,24 @@
+package expo.modules.kotlin.jni
+
+private var nextValue = 0
+
+private fun nextValue(): Int {
+  val result = 1 shl nextValue
+  nextValue++
+  return result
+}
```

Comment:
- Will this be compile-time optimized? 🤔 

P.S. 😜 
```suggestion
private var nextValue = 0

private fun nextValue(): Int = (1 shl nextValue).also { nextValue++ }
```